helpers/env-variables.ts

```diff
@@ -105,25 +104,6 @@ const getDataSourceEnvs = (
   llamaCloudKey?: string,
 ) => {
   switch (dataSource.type) {
```

Comment:
as we're moving the datasource config now to json with this PR, it's confusing to keep this code. 
how about generating `LLAMA_CLOUD_API_KEY` all the time? it simplifies the code and should not hurt

Reply:
Ah yes, i removed the env code, looks like I added it back when rebasing.